examples/script_nifti.py

```diff
@@ -0,0 +1,10 @@
+from shimmingtoolbox.dicom_to_nifti import dicom_to_nifti
+
+def main():
+
+    unsortedDicomDir = r"C:\Users\gaspa\Desktop\Test\dicom_unsorted"
```

Comment:
i would immediately start with an available example dataset (why doing it later when you can do it now-- it enables the team to follow your devs and reproduce your results)